build.gradle.kts

```diff
@@ -66,4 +66,7 @@ subprojects {
         targetCompatibility = JavaVersion.current()
         withSourcesJar()
     }
+    tasks.withType<JavaCompile>().configureEach {
+        options.isIncremental = true
```

Comment:
```suggestion
        options.isIncremental = true
        options.isFork = true
        options.encoding = "UTF-8"

Reply:
makes sense? though wouldnt you want ASCII encoding?